modules/monitoring-configuring-notifications-for-default-platform-alerts.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-notifications-for-default-platform-alerts_{context}"]
+= Configuring notifications for default platform alerts
+
+You can configure Alertmanager to send notifications. Customize where and how Alertmanager sends notifications about default platform alerts by editing the default configuration in the `alertmanager-main` secret in the `openshift-monitoring` namespace.
+
+[IMPORTANT]
+====
+Alertmanager does not send notifications by default. It is recommended to configure Alertmanager to receive notifications by setting up notifications details in the `alertmanager-main` secret configuration file.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+
+.Procedure
+
+. Open the Alertmanager YAML configuration file:
+
+** To open the Alertmanager configuration from the CLI:
+
+.. Print the currently active Alertmanager configuration from the `alertmanager-main` secret into `alertmanager.yaml` file:
```

Comment:
Still missing a "the", but I won't hold up the merge over it. But if you can address it in a future PR that'd be great :)

Reply:
Oh my, I apologise, I have no idea how I missed it even after checking it a few times :frowning: I will fix it in the next PR dealing with these sections :+1: